<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chattable</class>
 <widget class="QWidget" name="chattable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OurChat</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>-10</y>
     <width>20</width>
     <height>701</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>-10</y>
     <width>20</width>
     <height>701</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>841</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>530</y>
     <width>561</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="push_returnlogin">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>660</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:none;
background:transparent;
color: rgb(225, 225, 225);</string>
   </property>
   <property name="text">
    <string>点击返回</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="inputtext">
   <property name="geometry">
    <rect>
     <x>335</x>
     <y>576</y>
     <width>501</width>
     <height>65</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:none;</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:15pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:13pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendButton">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>654</y>
     <width>69</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(248, 248, 248);</string>
   </property>
   <property name="text">
    <string>发送(S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Return</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="search_user">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>25</y>
     <width>181</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>该功能暂未实现</string>
   </property>
  </widget>
  <widget class="QPushButton" name="search_pushButton">
   <property name="geometry">
    <rect>
     <x>265</x>
     <y>25</y>
     <width>26</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>20</y>
     <width>36</width>
     <height>36</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>qq.jpg</normaloff>qq.jpg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="exit_pushButton">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>0</y>
     <width>41</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新宋体</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:none;
background:transparent;</string>
   </property>
   <property name="text">
    <string>×</string>
   </property>
  </widget>
  <widget class="QPushButton" name="background_button">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>61</width>
     <height>691</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(42, 42, 42);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableWidget" name="online_tableWidget">
   <property name="geometry">
    <rect>
     <x>59</x>
     <y>88</y>
     <width>251</width>
     <height>600</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(231, 231, 231);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>249</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(231, 231, 231);</string>
   </property>
   <property name="text">
    <string> 在线用户</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>530</y>
     <width>561</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:none;
background-color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="check_pushButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>65</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>更新在线用户</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="tonameshow_textEdit">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>25</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:none;
background:transparent;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="chat_textBrowser">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>60</y>
     <width>550</width>
     <height>481</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(247, 247, 247);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_chat">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>80</y>
     <width>37</width>
     <height>37</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>select1.png</normaloff>select1.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>43</width>
     <height>43</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="save_message_button">
   <property name="geometry">
    <rect>
     <x>435</x>
     <y>545</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Message.png</normaloff>Message.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="picture_button">
   <property name="geometry">
    <rect>
     <x>335</x>
     <y>545</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Picture.png</normaloff>Picture.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="file_button">
   <property name="geometry">
    <rect>
     <x>368</x>
     <y>545</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>File.png</normaloff>File.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="cut_button">
   <property name="geometry">
    <rect>
     <x>402</x>
     <y>545</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Cut.png</normaloff>Cut.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <zorder>pushButton_2</zorder>
  <zorder>background_button</zorder>
  <zorder>line</zorder>
  <zorder>line_2</zorder>
  <zorder>line_3</zorder>
  <zorder>line_4</zorder>
  <zorder>push_returnlogin</zorder>
  <zorder>inputtext</zorder>
  <zorder>sendButton</zorder>
  <zorder>search_user</zorder>
  <zorder>search_pushButton</zorder>
  <zorder>pushButton</zorder>
  <zorder>exit_pushButton</zorder>
  <zorder>online_tableWidget</zorder>
  <zorder>label</zorder>
  <zorder>check_pushButton</zorder>
  <zorder>tonameshow_textEdit</zorder>
  <zorder>chat_textBrowser</zorder>
  <zorder>pushButton_chat</zorder>
  <zorder>save_message_button</zorder>
  <zorder>picture_button</zorder>
  <zorder>file_button</zorder>
  <zorder>cut_button</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
